### ISSUE TITLE
New Forwards for Developers

### ISSUE THREAD

**Issue by anaether** [sbpp/sourcebans-pp#473]
Is your feature request related to a problem? Please describe.
-------------------------------------------------------------

My idea is to make a forward when a player that is in the sourcebans list (and banned) when he got kicked that a forward is fired.

Describe the solution you'd like
--------------------------------

Idea would be to add to sourcebanspp.inc a new forward that will fired when a player get blocked by server to join because of his ban.

Additional context
------------------

I need it for a plugin that i code and it would be very usefull i think if you can add it pls to the source.

**Comment by rumblefrog** [comment]
Please use the template correctly.

**Comment by anaether** [comment]
Okey i looked again and maybe i found it. Ist this forward? SBPP_OnReportPlayer

**Comment by anaether** [comment]
Another Question is which effect will have !report from sbpp_report.sp. I have tryed on a player but in sourcebans panel logged in nothing and ingame same. The only thing that i see is that a db entry is created in sb_submission.

**Comment by rumblefrog** [comment]
You going to need to elaborate

**Comment by anaether** [comment]
Ok im already working on it. I will make then a pr.

**Comment by Groruk** [comment]
Reports normally show up in the webpanel under `Admin Panel` > `Bans` > `Ban submissions` (URL: `{domain.tld}/{dir to sbpp}/index.php?p=admin&c=bans#^2`).

And I don't really get the point of an `SBPP_OnPlayerKick` forward (maybe for statistics or a chat message but an forward for that would be a little overkill).

**Comment by anaether** [comment]
No Im writing a discord plugin with many integractions and i also want that the players that have been kicked by servers stands there 🍡

**Comment by anaether** [comment]
I created now a new PullRequest with my changes that works for me on my csgo server with sm 1.9.2 dev

**Comment by anaether** [comment]
Why closed stuff is already changed?

**Comment by rumblefrog** [comment]
Your PR was superseded by another, and there's no easy way for me to cherry-pick any of the commits, so this is now being handled by https://github.com/sbpp/discord-forward/issues/3
